front/src/core-logic/domain/establishmentLead/establishmentLead.epics.ts

```diff
@@ -0,0 +1,36 @@
+import { filter, map, switchMap } from "rxjs";
+import { establishmentLeadSlice } from "src/core-logic/domain/establishmentLead/establishmentLead.slice";
+import { catchEpicError } from "src/core-logic/storeConfig/catchEpicError";
+import {
+  ActionOfSlice,
+  AppEpic,
+} from "src/core-logic/storeConfig/redux.helpers";
+
+type EstablishmentLeadAction = ActionOfSlice<typeof establishmentLeadSlice>;
+
+const createEstablishmentLeadEpic: AppEpic<EstablishmentLeadAction> = (
+  action$,
+  _,
+  { establishmentLeadGateway },
+) =>
+  action$.pipe(
+    filter(
+      establishmentLeadSlice.actions.unsubscribeEstablishmentLeadRequested
+        .match,
+    ),
+    switchMap((action) =>
+      establishmentLeadGateway.rejectEstablishmentLeadRegistration$(
+        action.payload,
+      ),
+    ),
+    map((_result) =>
```

Comment:
```suggestion
    map(() =>
```

Etant donnée que le retour est void on doit même pouvoir simplifier plus comme ça :

```ts
map(establishmentLeadSlice.actions.unsubscribeEstablishmentLeadSucceeded)
```